packages/report-server/src/reportBuilder/transform/transform.ts

```diff
@@ -8,6 +8,7 @@ import { transformBuilders } from './functions';
 import { aliases } from './aliases';
 
 type TransformParams = {
+  name: string;
```

Comment:
This isn't actually doing anything, but is quite handy for debugging purposes so I left it in 👍